iron-scroll-target-behavior.html

```diff
@@ -66,18 +66,25 @@
       // Remove lister to the current scroll target
       if (this._oldScrollTarget) {
         if (this._oldScrollTarget === this._doc) {
+        
           window.removeEventListener('scroll', this._boundScrollHandler);
+        
         } else if (this._oldScrollTarget.removeEventListener) {
+        
           this._oldScrollTarget.removeEventListener('scroll', this._boundScrollHandler);
+        
         }
         this._oldScrollTarget = null;
       }
       if (isAttached) {
         // Support element id references
-        if (typeof scrollTarget === 'string') {
+        if (scrollTarget === 'document') {
```

Comment:
Mind adding doc to `scrollTarget` property section about using `document` to refer to document as the scrolling region